src/workloads/sharding/BatchedUpdateOneWithoutShardKeyWithId.yml

```diff
@@ -0,0 +1,134 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+Description: |
+  Runs the batched updateOne writes of type WithoutShardKeyWithId with a batch size of 1000.
+
+  The workload consists of 3 phases:
+    1. Creating an empty sharded collection distributed across all shards in the cluster.
+    2. Populating the sharded collection with data.
+    3. Running updateOne operations of type WriteWithoutShardKeyWithId.
+
+  The inserted documents have the following form:
+
+      {_id: 10, oldKey: 20, newKey: 30, counter: 0, padding: 'random string of bytes ...'}
+
+  The collection is sharded on {oldKey: 'hashed'}.
```

Comment:
What are the important metrics for this workload?

Reply:
Added, let me know if it suffices or if I need to write their names.